### PR TITLE
Iteration 3 integration

### PR DESCRIPTION
**Pull request by DougWelchons**
here is the integration with the sales engine and sales analyst files, as well as the refactors to get our simplecov coverage up to 100%. let me know if you see any changes we should make.

### CODE REVIEW

File: test/sales_analyst_test.rb [hunk 1]
```diff
@@ -8,269 +8,192 @@
...
+
+  def test_it_can_return_standard_deviation
+    engine = SalesEngine.from_csv(@csv_data)
+    analyst = SalesAnalyst.new(engine)
+
+    assert_equal 18.06, analyst.average_invoices_per_day_standard_deviation
+  end
+
+  def test_invoice_paid_in_full_with_invoice_id_arg
+
```
Comment (Diana20920) [inline test/sales_analyst_test.rb]: Gotta love these.

File: test/sales_analyst_test.rb [hunk 2]
```diff
@@ -8,269 +8,192 @@
...
+    analyst = SalesAnalyst.new(engine)
+
+    days_of_the_week = [
+                        "Saturday",
+                        "Friday",
+                        "Wednesday",
+                        "Monday",
+                        "Sunday",
+                        "Tuesday",
+                        "Thursday"
```
Comment (avjohnston) [inline test/sales_analyst_test.rb]: Let's make these single quotes so it matches what we've been doing for the rest of our files.

**Review by Diana20920 — APPROVED** [review]
Single quotes, and an extra line at the start of a method. Not bad. 
Looking great!

**Comment by brisag** [comment]
Wow! All looks fantastic! Were pushing through!

File: test/sales_analyst_test.rb (hunk 1 above)
Comment (avjohnston) [reply test/sales_analyst_test.rb]: Let's take this line out as well.

**Comment by avjohnston** [comment]
Looks awesome except for a few tiny syntax fixtures! Let's get those changed and then merge!

**Comment by DougWelchons** [comment]
good catch. i went ahead and fixt both of those items.

**Pull request merged**